packages/react-search-ui-views/src/styles/themes/reference-ui/components/_paging.scss

```diff
@@ -2,14 +2,27 @@
 @include block("paging") {
   > li {
     border: none;
-    background: transparent;
```

Comment:
Still needs transparent, theres a slight white background thats being applied if removed.
<img width="264" alt="Screenshot 2024-04-02 at 13 20 13" src="https://github.com/elastic/search-ui/assets/49480/0643af97-ce27-4174-855f-3331046648ca">

Reply:
Good catch! I have updated the css to use transparent background as before.